contracts/core/EntryPoint.sol

```diff
@@ -345,27 +374,23 @@ contract EntryPoint is IEntryPoint, StakeManager {
         MemoryUserOp memory mUserOp = opInfo.mUserOp;
         address sender = mUserOp.sender;
         _createSenderIfNeeded(opIndex, opInfo, op.initCode);
+        address paymaster = mUserOp.paymaster;
         if (aggregator == SIMULATE_FIND_AGGREGATOR) {
             numberMarker();
 
-            if (sender.code.length == 0) {
-                // it would revert anyway. but give a meaningful message
-                revert FailedOp(0, address(0), "AA20 account not deployed");
+            // solhint-disable-next-line no-empty-blocks
+            try this._simulateFindAggregator(sender, paymaster) {}
+            catch Error(string memory revertReason) {
+                revert FailedOp(opIndex, address(0), revertReason);
```

Comment:
you lost the paymaster value in the 'paymaster not deployed'.
However, it is a bit redundant. I was actually thinking to remove it, since:
- we only report paymaster this way, not factory (or aggregator)
- the "AA1x/AA2x/AA3x" code reflect if it is a factory/account/paymaster, respectively
- we always pass "index" so it can be derived from the original UserOp.
So I think that's ok here.

Reply:
How did I lose it? The previously emitted was address(0)

Reply:
Anyway fixed that.